language-design.md

```diff
@@ -102,6 +365,72 @@ you do so? If not, why not?
 
 **Response**
 
+For the most part, these two experiences of natural languages are not at odds
+with each other.  We interpreted "natural-language replacements for some of
+the more abstruse syntax" to refer more to the vocabulary of a language (for
+example, using `REPEAT` instead of `FOR`).  The grammar would still be intact,
+and the readers would still easily recognize the text as a programming language.
+The following is an example of Quorum, the language which was found to be more
+intuitive than Perl:
+```
+number x = 175.3
+text y = "false"
+if x > 100 then
+    y = "true"
+    end
+else then
+    y = "still false"
+    end
+```
+[Stefik et al., 2011]
+This block resembles code that we are familiar with, but some of the braces and
+parentheses have been replaced with words.  On the other hand, here is a
+snippet of AppleScript (from The Ultimate Beginner's Guide To AppleScript,
+Josh Johnson, 2012):
+
+```
+[applescript]
+tell application "Mail"
+
+set theMessage to make new outgoing message with properties
```

Comment:
I really like how you included an example of AppleScript. This line also shows the extent to which natural language is incorporated in the language. I can definitely see how writing AppleScript could be confusing.